client/src/components/existing-request/existing-request-edit.vue

```diff
@@ -30,51 +30,50 @@
 </template>
 
 <script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator'
+import NewReqModule from '@/store/new-request-module'
+import MainContainer from '@/components/new-request/main-container.vue'
+import EntityCannotBeAutoAnalyzed from '@/components/new-request/submit-request/entity-cannot-be-auto-analyzed.vue'
+import NamesCapture from '@/components/common/names-capture.vue'
```

Comment:
This is just reordering in the same order they are used in the template (which makes it easier to inspect and verify).

Also I added the EntityCannotBeAutoAnalyzed import, which was missing (but maybe is never rendered?).

Also I deleted the Succcess import, which wasn't being used.

Reply:
I think you are right, not entirely sure it is rendered anywhere.

Reply:
It is.

It used to be the network-layer patch methods that displayed it. It is now the view-layer code that displays it. It's just another page (not a modal).

Reply:
ah interesting! Fair enough